diff -urN evas-1.7.0/src/modules/loaders/svg/evas_image_load_esvg.c evas-1.7.0-patched/src/modules/loaders/svg/evas_image_load_esvg.c
--- evas-1.7.0/src/modules/loaders/svg/evas_image_load_esvg.c	2012-07-18 20:01:18.000000000 +1100
+++ evas-1.7.0-patched/src/modules/loaders/svg/evas_image_load_esvg.c	2012-10-10 00:19:23.148045184 +1100
@@ -72,7 +72,7 @@
         return EINA_FALSE;
      }
 
-   e = esvg_parser_load(file, NULL, NULL);
+   e = esvg_parser_load(file);
    if (!e)
      {
         *error = EVAS_LOAD_ERROR_DOES_NOT_EXIST;
@@ -152,7 +152,7 @@
         return EINA_FALSE;
      }
 
-   e = esvg_parser_load(file, NULL, NULL);
+   e = esvg_parser_load(file);
    if (!e)
      {
         *error = EVAS_LOAD_ERROR_DOES_NOT_EXIST;
